src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -7867,22 +7871,22 @@ void WriteBEPSTable(EnergyPlusData &state)
             ort->gatherElecSurplusSold = gtElecSurplusSold;
         }
         // convert to GJ
-        ort->gatherPowerFuelFireGen /= largeConversionFactor;
-        ort->gatherPowerPV /= largeConversionFactor;
-        ort->gatherPowerWind /= largeConversionFactor;
-        ort->gatherPowerHTGeothermal /= largeConversionFactor;
-        ort->gatherPowerConversion /= largeConversionFactor;
-        ort->gatherElecProduced /= largeConversionFactor;
-        ort->gatherElecPurchased /= largeConversionFactor;
-        ort->gatherElecSurplusSold /= largeConversionFactor;
+        ort->gatherPowerFuelFireGen /= (largeConversionFactor / ipElectricityConversionFactor);
```

Comment:
The compiler is going to do this for you anyway, but for readability you should probably define `Real64 convFactor = ipElectricityConversionFactor / largeConversionFactor;`. At that point you could also multiply rather than divide which is both faster and higher bandwidth, i.e., the processor has more multipliers than dividers because multiplication is common and also each multiplier is faster than each divider.

Reply:
Thanks! A good point to use multiplication whenever possible.